src/njoy21/input/argument/parser/test/parser.test.cpp

```diff
@@ -0,0 +1,3 @@
+#define CATCH_CONFIG_MAIN
+
+#include "catch.hpp"
```

Comment:
Ah. I did see all those tests (for which I was quite excited), but I didn't make the connection.